mock/vmHostMock.go

```diff
@@ -78,6 +78,10 @@ func (host *VmHostMock) ExecuteOnDestContext(_ *vmcommon.ContractCallInput) (*vm
 	return nil, nil, nil
 }
 
```

Comment:
comments ?

Reply:
Added.

Reply:
Not yet - there is an upcoming PR that refactors the mocks and stubs: https://github.com/ElrondNetwork/arwen-wasm-vm/pull/205

Will make sure that comments will be added there.